utils/Makefile.am

```diff
@@ -41,4 +41,6 @@ endif
 # All subdirectories
 SUBDIRS=$(MIRROR_VFD_DIR) $(TESTUTIL_DIR) $(TOOLS_DIR)
 
+bin_SCRIPTS = subfiling_vfd/h5fuse.sh
```

Comment:
It looks like scripts dont get distributed by default, so this might need to be `dist_bin_SCRIPTS`:

https://www.gnu.org/software/automake/manual/html_node/Scripts.html

Reply:
Actually, since h5fuse is setup with AC_CONFIG_FILES, `bin_SCRIPTS` should work here